BlueprintUICommonControls/Sources/AttributedLabel.swift

```diff
@@ -674,6 +749,10 @@ extension AttributedLabel {
         }
 
         var boundingShape: BoundingShape {
+            container?.boundingShape(for: self) ?? calculateBoundingShape()
```

Comment:
I think we should cache in the local coordinate space and we should apply the `UIAccessibility.convertToScreenCoordinates` here rather than below. Similar to `UIAccessibilityElement.accessibilityFrameInContainerSpace`.